recipes-extended/images/pelux-image-update/arp/emmcsetup1.lua

```diff
@@ -81,5 +81,35 @@ end
 function postinst()
 	local out = "Post installed script called"
 
+	ret, out = cmdexec("mkdir -p /tmp/mountedsda2")
+	if (ret == false) then
```

Comment:
A general comment. It's good practice to do comparisons such as
IF (CONSTANT == VARIABLE)
i.e.
if (false == ret)
if (1 == j)
etc

Yes, good compilers will find and notify about potential errors like:
if (ret = false)
but in some cases the code will actually be easier/faster to read if the variable is put at the beginning.

Reply:
sure this will reduce accidental errors. Will fix that.